environment/aws/sgw_setup/cert/sg_key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Spider sense tingling.... should we be checking in private keys???

Reply:
This might set off some alerts, but this is purely for running Sync Gateway with TLS enabled.  I guess I could change this so that it gets generated freshly every time